src/commands/general/Stats.ts

```diff
@@ -149,11 +149,12 @@ export class Stats extends BaseCommand {
 
         const pts = await LoggerManager.getPoints((resMember as IResolvedMember).member);
         let description = `${EmojiConstants.TICKET_EMOJI} Points: ${pts}`;
-        if(stats.quotaPoints > 0) {
-            description +=`\nQuota Points: ${stats.quotaPoints}`;
+        const serverQuotaPoints = stats.quotaPoints.find(x => x.key === ctx.guild?.id);
+        if (serverQuotaPoints) {
+            description += `\nQuota Points: ${stats.quotaPoints}`;
```

Comment:
This is a certified `[object Object]` moment 

![image](https://user-images.githubusercontent.com/37031713/208279727-93db9a29-7593-4d4b-b4ca-1dca82969795.png)